Header header
float32 lat
float32 lon
float32 alt
float32 cog
float32 speed
